cephalopod/factory.py

```diff
@@ -42,7 +42,6 @@ def make_app():
         'SQLALCHEMY_TRACK_MODIFICATIONS': False,
         'MULTIPASS_LOGIN_SELECTOR_TEMPLATE': 'login_selector.html',
         'MULTIPASS_LOGIN_FORM_TEMPLATE': 'login_form.html',
-        'MULTIPASS_SUCCESS_ENDPOINT': 'auth.index'
```

Comment:
this should stay. see the logic in multipass to get the next url:

        try:
            return session.pop('_multipass_next_url')
        except KeyError:
            return url_for(current_app.config['MULTIPASS_SUCCESS_ENDPOINT'])